### PR TITLE
web speed test results tests and monitoring

### PR DESCRIPTION
**Pull request by FelixZilber**

### CODE REVIEW

File: test/e2e/.travis.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+addons:
+  chrome: stable
+install:
+- cd $TRAVIS_BUILD_DIR/test/e2e
+- npm install
+- npm run linter
+- npm run compile
+before_script:
+- npm run grid > /dev/null 2>&1 &  
+- sleep 10
```
Comment (eitanp461) [inline test/e2e/.travis.yml]: Why do you need Travis to sleep?

File: test/e2e/README.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+[Allure Reporter ](https://github.com/webdriverio/wdio-allure-reporter "Allure Reporter ") for nice tests result representation
+
+## Getting Started
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes.
+### Prerequisites
+
+[NodeJs](https://nodejs.org/en/ "NodeJs") runtime engine
+[Chrome](https://www.google.com/chrome/ "Chrome")  for tests execution
+[FireFox](https://www.mozilla.org/en-US/ "FireFox")  for tests execution and display allure reporter
+[Java](http://www.oracle.com/technetwork/java/index.html "Java") for selenium grid and report generation
```
Comment (eitanp461) [inline test/e2e/README.md]: Any particular version?

File: test/e2e/README.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+[WebdriverIO](http://webdriver.io/ "WebdriverIO") as a base WebDriver implementation.
+[TypeScript](https://www.typescriptlang.org/ "TypeScript") for easier development process.
+[Mocha](https://mochajs.org/ "Mocha")  as a test runner
+[Allure Reporter ](https://github.com/webdriverio/wdio-allure-reporter "Allure Reporter ") for nice tests result representation
+
+## Getting Started
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes.
+### Prerequisites
+
+[NodeJs](https://nodejs.org/en/ "NodeJs") runtime engine
```
Comment (eitanp461) [inline test/e2e/README.md]: Any particular version? Latest? LTS?

File: test/e2e/README.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+[Chrome](https://www.google.com/chrome/ "Chrome")  for tests execution
+[FireFox](https://www.mozilla.org/en-US/ "FireFox")  for tests execution and display allure reporter
+[Java](http://www.oracle.com/technetwork/java/index.html "Java") for selenium grid and report generation 
+
+------------
+**Steps:**
+**Install packages:**
+> npm install
+
+**Compile:**
```
Comment (eitanp461) [inline test/e2e/README.md]: `Compile TypeScript`

File: test/e2e/package.json [hunk 5]
```diff
@@ -0,0 +1,58 @@
+{
+  "private": true,
+  "name": "web-speed-test-wdio",
+  "version": "0.0.1",
+  "description": "Web speed test using Mocha, WebDriverIO, Allure report ",
+  "author": "Felix Zilber",
+  "license": "MIT",
+  "engines": {
+    "node": ">7.5.0"
```
Comment (eitanp461) [inline test/e2e/package.json]: Does not match your `travis.yml` settings

File: test/e2e/package.json [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+    "mocha": "^5.0.5",
+    "prettier": "^1.11.1",
+    "tslint": "^5.9.1",
+    "tslint-config-airbnb": "^5.7.0",
+    "tslint-config-prettier": "^1.9.0",
+    "tslint-eslint-rules": "^5.1.0",
+    "tslint-microsoft-contrib": "^5.0.3",
+    "typescript": "^2.7.2",
+    "wdio-allure-reporter": "^0.6.0",
+    "wdio-allure-ts": "git+https://github.com/cloudinary/wdio-allure-ts.git",
```
Comment (eitanp461) [inline test/e2e/package.json]: Isn't this published as an NPM module?

File: test/e2e/wdio.conf.js [hunk 7]
```diff
@@ -0,0 +1,310 @@
...
+  /**
+   * Gets executed once before all workers get launched.
+   * @param {Object} config wdio configuration object
+   * @param {Array.<Object>} capabilities list of capabilities details
+   */
+  onPrepare: function(config, capabilities) {
+    /**
+     * Load configurations from .env
+     */
+    dotenv.config();
```
Comment (eitanp461) [inline test/e2e/wdio.conf.js]: Which configurations do you expect to read from `.env`? Anything worth mentioning in README?

File: test/e2e/.travis.yml (hunk 1 above)
Comment (felzilber) [reply test/e2e/.travis.yml]: In order to give for selenium grid to start

File: test/e2e/README.md (hunk 2 above)
Comment (FelixZilber) [reply test/e2e/README.md]: Readme updated

File: test/e2e/README.md (hunk 3 above)
Comment (FelixZilber) [reply test/e2e/README.md]: Readme updated

File: test/e2e/README.md (hunk 4 above)
Comment (FelixZilber) [reply test/e2e/README.md]: Readme updated

File: test/e2e/package.json (hunk 5 above)
Comment (FelixZilber) [reply test/e2e/package.json]: updated

File: test/e2e/package.json (hunk 6 above)
Comment (FelixZilber) [reply test/e2e/package.json]: not yet

File: test/e2e/wdio.conf.js (hunk 7 above)
Comment (FelixZilber) [reply test/e2e/wdio.conf.js]: removed. No use of .env in this project

**Pull request merged**